satpy/readers/ahi_hsd.py

```diff
@@ -583,6 +612,15 @@ def convert_to_radiance(self, data):
         else:
             gain = self._header["block5"]["gain_count2rad_conversion"][0]
             offset = self._header["block5"]["offset_count2rad_conversion"][0]
+        # If using custom calibration from GSICS, apply it here
+        if self.calib_mode == "CUSTOM":
+            if bname in self.custom_calib:
+                g2 = self.custom_calib[bname][0]
```

Comment:
Please use a more readable name for this variable, and the next one.

Reply:
Done!